h/models/user.py

```diff
@@ -117,7 +113,7 @@ def in_(self, userids):  # pylint: disable=arguments-renamed
         if not others:
             return False
 
-        return self.__clause_element__().in_(others)
+        return self.expression.in_(others)
```

Comment:
Yes. This very much. I found this in my travels, but it's totally off topic. This makes it much easier to understand in combo with the parent class.